nipype/interfaces/base/specs.py

```diff
@@ -165,29 +151,6 @@ def _deprecated_warn(self, obj, name, old, new):
                             '%s' % trait_spec.new_name: new
                         })
 
-    def _hash_infile(self, adict, key):
```

Comment:
Unused private method. Good to remove, as far as I'm concerned. Again, @satra, feel free to override if this looks like a mistake.